apps/token-manager/app/src/utils.js

```diff
@@ -1,5 +1,79 @@
 import BN from 'bn.js'
 
+/**
+ * Get the whole and decimal parts from a number.
+ * Trims leading and trailing zeroes.
+ *
+ * @param {string} num the number
+ * @returns {Array<string>} array with the [<whole>, <decimal>] parts of the number
+ */
+function splitDecimalNumber(num) {
```

Comment:
We should start moving these math and web3 utils somewhere else. Having them in Aragon UI could work, but they might be useful in other places too (e.g. CLI) and it would be weird to import Aragon UI only for that. Should we create a new package, like `aragon-utils` or similar? @izqui @sohkai

Reply:
I was thinking for a while of moving them all to a shared lerna package under aragon-apps, since it's mostly used here.

Let's do a good job of that later, so we can add tests and etc. :).